ssv/partial_sig_container.go

```diff
@@ -3,86 +3,117 @@ package ssv
 import (
 	"encoding/hex"
 
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+
 	"github.com/pkg/errors"
 	"github.com/ssvlabs/ssv-spec/types"
 )
 
+type SigningRoot string
```

Comment:
can be replaced to [32]byte
and then we dont need to hexify
(not a blocker for approval)